_posts/2023-07-19-trino-fest-2023-stripe.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: post
+title: "Inspecting Trino on ice"
+author: "Kevin Liu, Ryan Duan"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-fest-2023/Stripe.png
+show_pagenav: false
+---
+
+Stripe is an online payment processor that facilitates online payments for
+digital native merchants. They use Trino to facilitate ad hoc analytics, enable
```

Comment:
```suggestion
digital-native merchants. They use Trino to facilitate ad hoc analytics, enable
```
Maybe start with "For those unfamiliar" or something like that. Stripe is a huge company, most people will know what it is, and acknowledging that is sensible.